projects/hslayers/components/trip-planner/route-profile-selector.component.html

```diff
@@ -0,0 +1,21 @@
+<div class="flex-row w-100 m-auto justify-content-center align-items-center" style="display: flex">
+  <p class="m-0 p-0 flex-fill text-primary">{{'TRIP_PLANNER.selectProfile' | translateHs}}:</p>
+  <div ngbDropdown placement="bottom" style="max-width: 50%">
+    <button type="button" ngbDropdownToggle
+      class="btn btn-sm rounded-0 hs-toolbar-button d-flex align-items-center mw-100" aria-haspopup="true"
+      (click)="profilesExpanded = !profilesExpanded"
+      [attr.aria-expanded]="profilesExpanded">
+      <div class="text-truncate" *ngIf="selectedProfile !== undefined">
```

Comment:
We could maybe use new control flow as per https://github.com/hslayers/hslayers-ng/issues/4905